src/ansys/dpf/core/_version.py

```diff
@@ -4,27 +4,31 @@
 
 # Nice string for the version
 __version__ = ".".join(map(str, version_info))
-min_server_version = "2.0"
 
-server_to_ansys_grpc_dpf_version = {
-    "1.0": "==0.2.2",
-    "2.0": "==0.3.0",
-    "3.0": ">=0.4.0",
-    "4.0": ">=0.5.0",
-    "5.0": ">=0.6.0",
-    "6.0": ">=0.7.0",
-    "7.0": ">=0.8dev",
-}
+# Minimal DPF server version supported
+min_server_version = "4.0"
 
-server_to_ansys_version = {
-    "1.0": "2021R1",
-    "2.0": "2021R2",
-    "3.0": "2022R1",
-    "4.0": "2022R2",
-    "5.0": "2023R1",
-    "6.0": "2023R2",
-    "6.1": "2023R2",
-    "6.2": "2023R2",
-    "7.0": "2024R1",
-    "7.1": "2024R1",
-}
+
+class ServerToAnsysVersion:
+    legacy_version_map = {
```

Comment:
why is that called legacy?

Reply:
@cbellot000 the legacy versioning logic, vs the new one following the Ansys product versioning, but I can change the name, no problem

Reply:
I think calling it legacy makes it clear it should not be changed.